src/pages/starter-kit/checkout/getting-started.md

```diff
@@ -31,42 +34,63 @@ You must install or have access to the following prerequisites to develop with t
 
 Before installing Commerce modules, ensure that you have the required credentials in `auth.json` with [access to the Adobe Commerce repository](https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/prerequisites/authentication-keys).
 
-- Install the Out-of-Process Payment Extensions (OOPE) module on Adobe Commerce
+### Install the Out-of-Process Payment Extensions (OOPE) module on Adobe Commerce
+
+To enable out-of-process payment methods in Commerce, install the `magento/module-out-of-process-payment-methods`. This module enables out-of-process payment functionalities.
+To install the module, run the following command using Composer:
```

Comment:
```suggestion

To install the module, run the following command using Composer:
```

Reply:
You should combine all the installs into a single install section and use bullets for each install description followed by the install command